livestream-compose-sample/src/main/java/io/getstream/livestream/compose/ui/YoutubeLiveStreamScaffold.kt

```diff
@@ -0,0 +1,84 @@
+package io.getstream.livestream.compose.ui
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.material.Icon
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.material.TopAppBar
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Info
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+import io.getstream.livestream.compose.R
+import io.getstream.livestream.compose.topBarBackground
+
+/**
+ * View component to add support for scaffold with title,
+ * top bar is always dark grey with white content.
+ *
+ * @param modifier - Modifier for styling.
+ * @param onBackPressed - Handler for when the user clicks back press.
+ * @param title - Title for the header.
+ * @param onInfoClickHandler - Click handler to handle clicks on Info action icon
+ * @param actions - Composable function to render actions on top bar.
+ * @param content - Composable function to render content below the Scaffold.
+ */
+@Composable
+fun YoutubeLiveStreamScaffold(
```

Comment:
As I mentioned above, this doesn't make much sense to me - we're creating a very complex component that's in fact just using a `Scaffold` underneath. 

This is not a component our users will use if they want to customize the UI. Especially because we hardcode the `topBar` and `bottomBar`  components.

It's better to just use a `Scaffold` at the call site.

Reply:
Ok , a general question on custom components for showcase app :

 This showcase is not a SDK code so why would users want to not use this as is ( consumer facing app per say ) 
I get the customizations part on `topBar` and any other internally inflated attrs which are not exposed.

Should we always prioritize simple call site components ?